staging/src/k8s.io/client-go/util/workqueue/delaying_queue.go

```diff
@@ -199,7 +199,7 @@ func (q *delayingType) waitingLoop() {
 	waitingEntryByData := map[t]*waitFor{}
 
 	for {
-		if q.Interface.ShuttingDown() {
+		if q.ShuttingDown() {
```

Comment:
As `q.Interface` is an embedded field, I think both the two ways to call  `ShuttingDown` are ok via https://golang.google.cn/doc/effective_go#embedding. So I wonder if it's necessary to make this change?  Correct me if I misunderstand.

Reply:
There is no difference between the two coding style at functional, but I think it is more harmonious to keep the style consistent; Other methods called for q do not have an Interface field:

- `q.Add(entry.data)` at line 216 245 255
- `q.ShuttingDown()` at line 162
- `q.Add(item)` at line 170

Reply:
If so, line 153 should be changed, too.

Reply:
I'll do it right now

Reply:
> If so, line 153 should be changed, too.

153 can't be changed for the same name method: `func (q *delayingType) ShutDown()`

Reply:
Oh，yeah. It's my negligence.

Reply:
Uh，yeah. It's my negligence.